.github/actions/constellation_create/action.yml

```
@@ -59,6 +59,9 @@ inputs:
   force:
     description: "Set the force-flag on apply to ignore version mismatches."
     required: false
+  encryption-secret:
+    description: 'The secret to use for encrypting the artifact.'
```

```suggestion
    description: "The secret to use for encrypting the artifact."
```
nit: I don't think we have a set style guide for the quoting style in our actions/workflows, but it should at least be kept consistent in one action.
Also applies to other actions changed by this PR